packages/ceramic-core/package.json

```diff
@@ -36,10 +36,12 @@
     "@ethersproject/base64": "^5.0.0-beta.134",
     "@ethersproject/providers": "^5.0.0-beta.144",
     "@ethersproject/random": "^5.0.0-beta.136",
+    "@textile/powergate-client": "^0.2.0",
```

Comment:
This dependency is quite large ([500K](https://bundlephobia.com/result?p=@textile/powergate-client@0.2.0)). In the future we should consider if the powergate pinning is an external package that is passed in as a module to ceramic-core. This way you could still run ceramic-core as a lightweight client. In the cli you would of course always include this package.